docs/_docs/wallet-extension/wallet-extension.md

```diff
@@ -75,8 +75,8 @@ tools.
    automatically by the wallet extension. Your balance in MetaMask will be set to 0 at the beginning.
    You can get testnet ETH from [faucet](https://docs.obscu.ro/testnet/faucet/) (you may need to switch to another network and back again to force MetaMask to refresh balance).
    
-   Once a viewing key is generated it will be persisted across restarts of the wallet extension, saved in the user home space under `~/.obscuro/wallet_extension_persistence`.
+   Once a viewing key is generated it will be persisted across restarts of the wallet extension, saved in the user home space under `~/.ten/wallet_extension_persistence`.
```

Comment:
Might want to leave this one so it can be found when it's updated but not sure

Reply:
good spot